Podverse/Podcast.swift

```diff
@@ -36,9 +34,18 @@ class Podcast: NSManagedObject {
         self.mutableSetValue(forKey: "episodes").remove(value)
     }
     
-    static func retrieveMediaRefsFromServer(episodeMediaUrl: String? = nil, podcastFeedUrl: String? = nil, onlySubscribed: Bool? = nil, completion: @escaping (_ mediaRefs:[MediaRef]?) -> Void) {
+    static func retrievePodcastByFeedUrl(feedUrlString: String) -> Podcast? {
```

Comment:
If we are planning on fetching the podcast by feedUrlString, could we not just fetch it using its managedObjectId? It is also a unique id, right? It seems like a better idea to always use data that we know for a fact will be there (moid) instead of a coreData object property that potentially is not there.